packages/core/src/connections/ConnectorInterface.ts

```diff
@@ -13,5 +13,5 @@ export interface ConnectorInterface {
   appsForOrg?(orgAddress: string): Promise<App[]>
   repoForApp?(appAddress: string): Promise<Repo>
   appByAddress?(appAddress: string): Promise<App>
-  roleById?(roleId: string): Promise<Role>
+  rolesByAddress?(appAddress: string): Promise<Role[]>
```

Comment:
I decided to change the interface to return all the roles by an App address. So if we want a single role we can filter the array.

Reply:
Should it be named `rolesForApp` to be more aligned with the others?

Reply:
Agree with Pierre!